ingest/molecules/moleculeMetadData.json

```diff
@@ -12,6 +12,7 @@
     },
     "data": { 
       "type": "array",
+      "minItems": 1,
```

Comment:
Noticed a typo in the filename "MetadData" should be just "MetaData". Otherwise ok.